fandogh_cli/service_commands.py

```diff
@@ -196,7 +196,27 @@ def service_apply(file, parameters):
         Managed service deployed successfully
         """
 
-    click.echo(message)
+    if detach:
+        click.echo(message)
+    else:
+        while True:
+            details = get_details(service_name)
+
+            if not details:
+                exit(1)
```

Comment:
why did you exit() here?what does it mean when get_details() returns None?

Reply:
Refer to other command ;)